### PR TITLE
PLT-333: Script for running a local node in buildkite

### PR DESCRIPTION
**Pull request by zliu41**
Test run: https://buildkite.com/input-output-hk/plutus-script-evaluation/builds/21#

### CODE REVIEW

File: scripts/cardano-node.sh [hunk 1]
```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
```
Comment (michaelpj) [inline scripts/cardano-node.sh:1]: Hmm, I wonder if there's a nix-based version of this we can use? I'm surprised if there isn't something like this somewhere else already, and that might require more straightforward maintenance than this one. Then again, I don't *actually* want to have a "real" dependency on the node, so I'm not sure what the best thing to do is. Maybe this is fine for now and we can worry about it if it becomes hard to maintain.

File: scripts/cardano-node.sh [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+tar zxvf "$NODE_BIN_DIR"/$node_zipped -C "$NODE_BIN_DIR"
+
+mkdir -p "$NODE_DIR"
+# Download config files
+for x in "${node_config_files[@]}"; do
+  wget -nc https://hydra.iohk.io/build/7370192/download/1/"$x" -P "$NODE_DIR"
+done
+
+# Launch node
```
Comment (michaelpj) [inline scripts/cardano-node.sh:36]: I think we should separate the downloading and running parts. There's no reason to download the node every single time if we're just going to run it on a machine that already has it.

**Review by michaelpj — APPROVED** [review]
FYI @zeme-iohk

File: scripts/cardano-node.sh (hunk 1 above)
Comment (zliu41) [reply scripts/cardano-node.sh:1]: Yeah, you can run the node by checking out cardano-node, but I agree having to depend on the cardano-node repo isn't ideal.

File: scripts/cardano-node.sh (hunk 2 above)
Comment (michaelpj) [reply scripts/cardano-node.sh:36]: Sounds like you need some comments :)

**Comment by zliu41** [comment]
Closed in favor of https://github.com/input-output-hk/plutus-apps/pull/531

**Pull request closed without merging**